ipet/Key.py

```diff
@@ -98,9 +98,9 @@ class SolverStatusCodes:
     TimeLimit = 2
     MemoryLimit = 3
     NodeLimit = 4
+    GapLimit = 4.5
```

Comment:
Thanks, that is for the sorting, right? Let's keep it integer. In order to keep some space, we can just have the non-limit solver codes starting from 10 or 100, or whatever.

Reply:
Correct. I am unsure about changing the existing values, wouldn't that mean that all data has to be parsed again? (I.e. the rubberband data)
If not then yes, i would also like to keep it integer.